src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/connection/ConnectionManager.java

```diff
@@ -566,12 +675,15 @@ private static class ChannelHolder {
         private final Supplier<Channel> supplier;
         private final BiConsumer<ShutdownNotifier, Boolean> reconnectionChecker;
         private final int maxCount;
+        private final SubscriptionCallbacks subscriptionCallbacks;
```

Comment:
Please rename to SubscriptionCallback because it contains only one call back now

Reply:
but it actually contains two callbacks: deliver callback and cancel callback

Reply:
OK, I missed this fact